src/pages/main/index.tsx

```diff
@@ -1,3 +1,68 @@
+import { api } from '@/api';
+import { useEffect, useState } from 'react';
+import { wrapper } from './index.styles';
+import { categoryList } from './index.styles';
+import { categoryItems } from './index.styles';
+import { categoryItem } from './index.styles';
+import { imgIcon } from './index.styles';
+import { useNavigate } from 'react-router-dom';
+
+interface ImageData {
+  id: number;
+  name: string;
+  icon: string;
+}
+
 export default function Main() {
-  return <main>Pream</main>;
+  const [imageData, setImageData] = useState<ImageData[]>([]);
+  const nav = useNavigate();
+
+  useEffect(() => {
+    const getCategories = async () => {
+      try {
+        const response = await api.get('/categories');
+        console.log('response: ', response);
+        setImageData(response.data);
+      } catch (error) {
+        console.error('failed: ', error);
+      }
+    };
+    getCategories();
+  }, []);
+
+  const handleCategoryButtonClick = (link: number) => {
+    nav(link);
+  };
+
+  return (
+    <div css={wrapper}>
+      <header>header</header>
+      <main>
+        <div>carousel</div>
+        <div css={categoryList}>
+          <div css={categoryItems}>
+            {imageData.length > 0 ? (
+              imageData
+                .filter((item) => item.id !== 4 && item.id !== 5)
+                .map((item) => (
+                  <div key={item.id} css={categoryItem}>
+                    <img
+                      src={item.icon}
+                      alt={item.name}
+                      css={imgIcon}
+                      onClick={() => handleCategoryButtonClick(item.id)}
```

Comment:
`handleCategoryButtonClick` 함수가 짧아서 별도로 분리 안 하고 이렇게 여기서 바로 써도 될 것 같아요!
```suggestion
                      onClick={() => navigate(item.id)}
```

Reply:
오 바로 수정하겠습니다